src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
```

Comment:
You don't need to declare an empty object before assigning a value to 'tempKyesRobot' and 'tempValueRobot'. You can directly assign the values.